extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -62,10 +67,18 @@ public void build(ApplicationInfoBuildItem applicationInfo,
             throw new RuntimeException("Unable to setup environment for generating Kubernetes resources", e);
         }
 
+        ApplicationPropertiesConfigSource applicationProperties = new ApplicationPropertiesConfigSource.InJar();
+        Map<String, Object> config = applicationProperties.getProperties().entrySet()
+                .stream()
+                .filter(e -> e.getKey().startsWith("dekorate"))
+                .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
+
         final SessionWriter sessionWriter = new SimpleFileWriter(root, false);
-        sessionWriter.setProject(new Project());
+        Project project = FileProjectFactory.create(new File("."));
```

Comment:
BTW, a more elegant way of getting any file under the module dir, would help.

Reply:
What do you have in mind?

Reply:
All I need is the path to ANY file under the module dir (e.g. path to pom.xml, application.properties or even the path to the module dir itself).
I feel that `new File(".") is not ideal.

Reply:
I am not aware of anything that would give you this information